dist/install.sh

```diff
@@ -142,14 +157,14 @@ download_tarball() {
 }
 
 if download_tarball; then
-    say_white "+ Extracting to $HOME/.pulumi/bin"
+    say_white "+ Extracting to ${PULUMI_INSTALL_ROOT}/bin"
 
-    # If `~/.pulumi/bin` exists, remove previous files with a pulumi prefix
-    if [ -e "${HOME}/.pulumi/bin/pulumi" ]; then
-        rm "${HOME}/.pulumi/bin"/pulumi*
+    # If \`~/.pulumi/bin\` exists, remove previous files with a pulumi prefix
```

Comment:
```suggestion
    # If `~/.pulumi/bin` exists, remove previous files with a pulumi prefix
```

I don't think the backslashes here are necessary.